Sources/EpoxyCollectionView/Models/SupplementaryItemModel/SupplementaryItemModeling.swift

```diff
@@ -36,6 +36,13 @@ protocol InternalSupplementaryItemModeling: SupplementaryItemModeling,
     traitCollection: UITraitCollection,
     animated: Bool)
 
+  /// Creates view for this supplementary item. This should only be used to create a view outside of a collection
```

Comment:
Seems good to have a symmetrical API to what we have in `ItemModeling` 👍  https://github.com/airbnb/epoxy-ios/blob/42d2ad2d0f7bc78594ea5ab1534c02a2fe257601/Sources/EpoxyCollectionView/Models/ItemModel/ItemModeling.swift#L45-L50